sp/src/game/server/hl2/npc_manhack.cpp

```diff
@@ -2973,6 +3019,32 @@ bool CNPC_Manhack::HandleInteraction(int interactionType, void* data, CBaseComba
 		return false;
 	}
 
+#ifdef EZ
+	if ( interactionType == g_interactionZombinePullGrenade )
+	{
+
+#ifdef EZ2
+		m_bNemesis = true;
```

Comment:
Is this so that the manhacks are still blue in E:Z2? It brings up a few questions about the lore :eyes:

Reply:
Hmm... You know, honestly I didn't contemplate how this would make the manhacks drawn by zombine would be friendly towards the Combine nemesis class. It was intended to make them hostile towards both Combine and rebels.

Reply:
Wouldn't they already be hostile due to the code in `Classify()`?

Reply:
Yes, but that's only while held. There's also a relationship applied to the manhack to make them hate the zombine's last target. But in the context of EZ2, I think it makes the most sense to make them 'wild' manhacks with their wires crossed. There isn't an explanation in the lore for why such manhacks don't attack the nemesis, even though that's why they exist.